airbyte-integrations/bases/standard-destination-test/src/main/resources/v1/exchange_rate_catalog.json

```diff
@@ -11,7 +11,7 @@
             "$ref": "WellKnownTypes.json#/definitions/String"
           },
           "date": {
-            "$ref": "WellKnownTypes.json#/definitions/Date"
```

Comment:
Long story short :). This dataset is used in a different tests (see `EXCHANGE_RATE_CONFIG` constant  in a `DestinationAcceptanceTest`). One of the issue that I faced was testLineBreakCharacters test which broke avro converter   (which is why I added https://github.com/airbytehq/json-avro-converter/pull/17/files#diff-c994d6172cabf7dca250e0806b69c501d475caac548cfd9a574e2458b7da3b5aR24 by the way 😃) and I was not sure what will happen with others connector's tests if I'll change it. 

I see two variants:
1. keep `"$ref": "WellKnownTypes.json#/definitions/Date"`. Here I'll need to be sure that all it passed for destination s3 V1. Then all who add V1 support need to be sure that connector supports different variants of date time with timezone
2. Temporary use it as string and create a ticket for this refactoring. (I didn't notice it during V1 tests adaptation)

WDYT?

Reply:
Agree with you but probably will need to revisit it. Since now it will a bit of pain to adapt it. 
The data types are tested with `data_type_object_test_catalog.json, data_type_array_test_catalog.json, data_type_array_object_test_catalog.json` datasets